File: Java/Bioskop.java
//membuat class public dengan nama bioskop beserta atribut nya
public class Bioskop {
    private int id;
    private String nama;
    private String alamat;
    private int jumlah_studio;
    private String kota;
    
    //sebuah konstruk untuk class Bioskop
    Bioskop(){

    }
    //konstruk juga tetapi dengan meminta masukan juga agar data bisa langsung terisi
    Bioskop(int id, String nama, String alamat, int jumlah_studio, String kota){
        this.id = id;
        this.nama = nama;
        this.alamat = alamat;
        this.jumlah_studio = jumlah_studio;
        this.kota = kota;
    }
    //semua method untuk mendapatkan data dari class dan memasukan data ke class
    //set untuk memasukan data, get untuk mengambil data
    void setid(int id){
        this.id = id;
    }
    int getid(){
        return this.id;
    }
    void setnama(String nama){
        this.nama = nama;
    }
    String getnama(){
        return this.nama;
    }
    void setalamat(String alamat){
        this.alamat = alamat;
    }
    String getalamat(){
        return this.alamat;
    }
    void setjumlah_studio(int jumlah_studio){
        this.jumlah_studio = jumlah_studio;
    }
    int getjumlah_studio(){
        return this.jumlah_studio;
    }
    void setkota(String kota){
        this.kota = kota;
    }
    String getkota(){
        return this.kota;
    }

}

File: PHP/Bioskop.php
<?php

    class Bioskop{
        private int $id;
        private string $nama;
        private string $alamat;
        private int $jumlah_studio;
        private string $kota;
        private ?string $gambar;
        public function __construct(int $id, string $nama, string $alamat, int $jumlah_studio, string $kota, ?string $gambar = null){
            $this->id = $id;
            $this->nama = $nama;
            $this->alamat = $alamat;
            $this->jumlah_studio = $jumlah_studio;
            $this->kota = $kota;
            $this->gambar = $gambar;
        }
        public function getid(): int{
            return $this->id;
        }
        public function setid(int $id): void{
            $this->id = $id;
        }
        public function getnama(): string{
            return $this->nama;
        }
        public function setnama(string $nama): void{
            $this->nama = $nama;
        }
        public function getalamat(): string{
            return $this->alamat;
        }
        public function setalamat(string $alamat): void{
            $this->alamat = $alamat;
        }
        public function getjumlah_studio(): int{
            return $this->jumlah_studio;
        }
        public function setjumlah_studio(int $jumlah_studio): void{
            $this->jumlah_studio = $jumlah_studio;
        }
        public function getkota(): string{
            return $this->kota;
        }
        public function setkota(string $kota): void{
            $this->kota = $kota;
        }
        public function getgambar(): ?string{
            return $this->gambar;
        }
        public function setgambar(?string $gambar): void{
            $this->gambar = $gambar;
        }
        
    }


?>